ext/drreg/drreg.c

```diff
@@ -68,14 +68,6 @@
 /* This should be pretty hard to exceed as there aren't this many GPRs */
 #define MAX_SPILLS (SPILL_SLOT_MAX + 8)
 
-/* We store data about spill slot usage in the data area of a label instr. */
-#define SPILL_SLOT_ID_LABEL_INSTR_DATA_AREA_OFFSET 0
```

Comment:
This was separately removed: looks like you merged w/ master, and the "View changes since last review" is showing these unrelated changes?  It used to be that nearly all code review systems would do that and it was best to never merge until after the final review was done; in the just the last couple of years code review systems got better at hiding unrelated changes, but that seems to have failed here...not sure how to get a view of just new changes -- do you see a way to do it?

Reply:
I merged early this time because I wanted to test with changes from a related master commit. But I probably should have done that locally and waited for the review to complete before pushing it.

> not sure how to get a view of just new changes -- do you see a way to do it?

Only idea I have is looking at individual commits from the last review. But nothing that would put all changes in one view, minus the master merge noise.